src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -37,6 +37,14 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		self.store = store
 		self.client = client
 		self.configurationProvider = appConfig
+		
+		self.configurationProvider.appConfiguration().sink { [ weak self] configuration in
```

Comment:
I'm concerned, that this sink is only executed once.
The config data could get stale.
Why not executing it every time triggerSubmitData is called like before? (nil check ("if cancellable == nil") should be removed nevertheless)

Reply:
I believe this sink will always be called because there is a strong reference to the configurationProvider.

Reply:
i will have a look on your considerations and will reopen my PR because now I think the original problem was another :(